src/modules/job-manager/plugins/perilog.c

```diff
@@ -157,7 +157,6 @@ static struct perilog_procdesc *perilog_procdesc_create (json_t *o,
 {
     struct perilog_procdesc *pd = NULL;
```

Comment:
Suggestion: squash these consecutive commits down to one or two:
- 6a61e3a6a8e44c190faa85ab9eee7103d777869b perilog: add use-imp config parameter
- ad9f0c1b657c222ed74993ea540172536e88dbfe perilog: improve error message on failure
- ff1d8a6a3821490532a9860022a021dc7c4252dc  perilog: remove use-imp config

Reply:
Thanks for catching that, I had forgotten that incremental modification was in there.